jest-native-config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
-  testMatch: ["**/jest/**/?(*.)+(spec|test).[jt]s?(x)"],
+  testMatch: ["**/jest/victory-native/**/?(*.)+(spec|test).[jt]s?(x)"],
```

Comment:
We might want to move these tests to a completely separate folder, but for now I just configured the jest native tests to run tests only in this folder, and added an exclude path option to `jest-config.js` to exclude this folder.